Source/Core/Core/HW/AudioInterface.cpp

```diff
@@ -89,7 +89,7 @@ void AudioInterfaceManager::DoState(PointerWrap& p)
   p.Do(m_aid_sample_rate_divisor);
   p.Do(m_cpu_cycles_per_sample);
 
-  SoundStream* sound_stream = m_system.GetSoundStream();
+  const SoundStream* sound_stream = m_system.GetSoundStream();
```

Comment:
To silence your `const` warnings, I wonder if it makes more sense to instead make `SoundStream::GetMixer()` non-const, since it's returning a pointer to non-const `Mixer`.

I don't think it necessarily makes sense for a `const` getter to return a pointer/reference to non-const.
I guess all of our getters in `System` behave like this, though I don't know if I care for that.

Others can comment.